.github/workflows/platformDeploy.yml

```diff
@@ -31,40 +31,12 @@ jobs:
     if: ${{ fromJSON(needs.validateActor.outputs.IS_DEPLOYER) }}
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: Expensify/App/.github/actions/composite/setupNode@main
 
-      - uses: actions/setup-ruby@v1
+      - uses: ruby/setup-ruby@08245253a76fa4d1e459b7809579c62bd9eb718a
         with:
           ruby-version: '2.7'
-
-      - uses: actions/cache@v2
-        with:
-          path: vendor/bundle
-          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
-
-      - name: Install bundler
-        run: |
-          bundle config path vendor/bundle
```

Comment:
From the [ruby/setup-ruby docs](https://github.com/ruby/setup-ruby#bundle-config):

> To perform caching, this action will use bundle config --local path $PWD/vendor/bundle.
Therefore, the Bundler path should not be changed in your workflow for the cache to work (no bundle config path).

Since the action already used `vendor/bundle` for the path, we don't need to worry about setting it before installing bundler.